src/backend/base/langflow/components/nvidia/nemo_evaluator.py

```diff
@@ -191,26 +167,121 @@ class NVIDIANeMoEvaluatorComponent(Component):
             min_label="Precise",
             max_label="Creative",
             value=0.1,
-            info="The temperature to be used during generation sampling (0.0 to 2.0).",
+            info="The temperature to be used during generation sampling (0.0 to 2.0)."
         ),
-        IntInput(name="313_top_k", display_name="Top_k", info="Top_k value for generation sampling.", value=1),
-        # Conditional input for run_inference = False
-        StrInput(
-            name="320_output_file",
-            display_name="Output File",
-            info="Path of the output file in NeMo Data Store in nds:<dataset>/<json output file>.",
+        IntInput(
+            name="313_top_k",
+            display_name="Top_k",
+            info="Top_k value for generation sampling.",
+            value=1
+        ),
+        MessageTextInput(
```

Comment:
I don't understand why these inputs Dataset and Training Data are being added to the list of options for LM Eval Harness. I do not see them in the API. https://developer.nvidia.com/docs/nemo-microservices/evaluation/source/evaluations/evaluations_llm.html#lm-evaluation-harness